operations/README.md

```diff
@@ -2,6 +2,12 @@
 
 This section describes all operations supported by the API, organised here by theme.
 
+## Reservation groups
```

Comment:
This should be added under ## Reservations, no need to create a new section :-). Just to be clear, the operations are grouped together into themes on this pages, it's not one section per operation or else there would be a hundred sections.